website/docs/r/cm_offering_instance.html.markdown

```diff
@@ -34,10 +34,12 @@ Review the argument reference that you can specify for your resource.
 - `cluster_region` - (Required, String) The cluster region for example, `us-south`.
```

Comment:
Please add timeout block in docs
Please refer
https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/container_vpc_cluster#timeouts